### ISSUE TITLE
"Span" prefix is unnecessary verbose

### ISSUE THREAD

**Issue by Multirious** [Multirious/bevy_tween#11]
At early development, it's designed specifically that you can implement custom tweener and so the "Span" prefix is added to the default tweener in this crate to prevent confusion with possibly user custom tweener. But at later development I've realized that most people won't necessary need a custom tweener. "Span" prefix should be stripped out by default to reduce verbosity.

**Comment by musjj** [comment]
Speaking of verbosity, there might be some lessons that could be learned from https://github.com/nixon-voxell/bevy_motiongfx (and Motion Canvas). I really like how simple and brief the API is:
```rust
chain(&[
    all(&[
        commands.play(
            line_motion
                .transform
                .translate_add(Vec3::new(0.0, -100.0, 0.0)),
            1.5,
        ),
        commands.play(line_motion.line.extend(100.0), 1.0),
        commands.play(line_motion.stroke.style_to(10.0), 1.0),
    ]),
    all(&[
        commands.play(
            line_motion
                .transform
                .translate_add(Vec3::new(0.0, 100.0, 0.0)),
            1.5,
        ),
        commands.play(line_motion.line.extend(-100.0), 1.0),
        commands.play(line_motion.stroke.style_to(1.0), 1.0),
    ]),
])
```
So little unnecessary noise which makes constructing complex and long animations a lot easier.

But after using the plugin for a while, it very much feels that it's optimized for presentation rather than game and interactivity, which is why I ended up moving to `bevy_tween`.

**Comment by Multirious** [comment]
Yeah, the current builder API right now is made to help with boilerplates *but* prioritizes flexibility. Some more adjustments or additions could be made. I'd like further comments and ideas.
